google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -882,7 +882,9 @@ public void write(Iterable<LogEntry> logEntries, WriteOption... options) {
         options = Instrumentation.addPartialSuccessOption(options);
       }
       writeLogEntries(logEntries, options);
-      if (flushSeverity != null) {
+      // We treat Severity.UNRECOGNIZED as an option to disable flushing completely,
+      // since LogSeverity does not exposed OFF as a severity level
+      if (flushSeverity != null && flushSeverity != Severity.UNRECOGNIZED) {
```

Comment:
The implementation tries to use `enum Severity` for more than one goal: to define the severity level **and** to define on/off conditions for flushing. I do not think it is a good idea.

Reply:
I guess `Level` does the same - it combines `OFF` with other levels...